### PR TITLE
Add best feed type

### PR DESCRIPTION
**Pull request by matthiaskern**

### CODE REVIEW

**Comment by matthiaskern** [comment]
Sure, I'll try to add that!

**Comment by matthiaskern** [comment]
@clintonwoo I added the behaviour with a conditional, is that alright?

File: src/components/presentational/HeaderNav.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import Link from 'next/link';
 import PropTypes from 'prop-types';
 
-const HeaderNav = props => (
+const HeaderNav = props => (console.log(props.currentUrl),
```
Comment (clintonwoo) [inline src/components/presentational/HeaderNav.js]: Did you mistakenly leave this line in? Looks good to me without this line

**Comment by matthiaskern** [comment]
@clintonwoo Whoops. Removed the log.

**Comment by clintonwoo** [comment]
Thanks. Good work

**Pull request merged**